src/main.js

```diff
@@ -209,11 +209,12 @@ function createPosterHtml(posterObj) {
 
 function deleteMiniPoster(event) {
   var deletedPosters = [];
-  var closestElement = event.target.closest(".mini-poster")
+  var closestElement = event.target.closest('.mini-poster")
```

Comment:
need to fix this quote mark to be the same but other wise good work

Reply:
Damn Sorry I missed that

Reply:
It's weird because they are the same in mine